safe/gis/generic_expressions.py

```diff
@@ -71,6 +73,46 @@ def inasafe_analysis_summary_field_value(field, feature, parent):
     return feature[index]
 
 
+description = tr('Retrieve a value from a field in the sub impact analysis '
+                 'layer from a multi exposure analysis layer.')
+examples = {
+    'inasafe_multi_exposure_field_value('
+    '\'population\', \'total_not_exposed\')': 3
+}
+help_message = generate_expression_help(description, examples)
+
+
+@qgsfunction(
+    args='auto', group='InaSAFE', usesGeometry=False, referencedColumns=[],
+    help_text=help_message.to_html(), helpText=help_message.to_html())
+def inasafe_multi_exposure_field_value(exposure_key, field, feature, parent):
```

Comment:
It's not clear that the expression if fetching the field in the analysis summary layer. Only by reading the code I understand it's fetching in the analysis summary layer (and not in the aggregation layer for instance) We need to update the expression name IMHO?
`inasafe_sub_analysis_summary_field_value`?

I know it's difficult to find a good name :D

Reply:
aggree :+1: